tools/bin/commands/util/openshift_funcs

```diff
@@ -1,11 +1,66 @@
 #!/bin/bash
 
+OC_MIN_VERSION=3.9.0
+
+compare_version_part() {
+    local test=$1
+    local min=$2
+
+    test=$(($test + 0))
+    min=$(($min + 0))
+
+    if [ $test -eq $min ]; then
+        return 0;
```

Comment:
Actually we are using a different semantics for returning values from functions, which can be used also for string return values.

It works with eval and echo:

```
a() {
   ...
   echo "myretval"
}

x=$(a)
```

Also we have golang like error handling:

```
a() {
    ...
    echo "ERROR: Can't do this"
}

x=$(a)
check_error $x
```

See also other spots in the scripts how this is used.
It would be awesome if we could stick to this convention, also for pure numeric return values.